### DISCUSSION TITLE
Laravel Repository Pattern

### DISCUSSION THREAD

**Discussion by salehhashemi1992** [HassanZahirnia/laravel-package-ocean#171] [Package Suggestions]
This Laravel package abstracts the database layer using repository pattern with enhanced capabilities for filtering and searching. It simplifies the common data manipulation tasks, along with advanced features for applying custom filters and search criteria.

https://github.com/salehhashemi1992/laravel-repository

**Comment by salehhashemi1992** [comment]
@HassanZahirnia

**Reply by HassanZahirnia** [reply]
Hey I tried to add your package but since I have strict validation rules set in place for checking the composer.json and making sure there is support for the latest Laravel versions, I couldn't simply do it. You have to make sure you provide support for the active Laravel versions.
